docs/resources/administration_org_policy.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: "spotinst"
+page_title: "Spotinst: health_check"
+subcategory: "Elastigroup"
+description: |-
+  Provides a Spotinst Health Check resource.
+---
+
+# spotinst\_administration\_org\_policy
+
+Provides a Spotinst access policy 
+
+## Example Usage
+
+```hcl 
+resource "spotinst_administration_org_policy" "terraform_policy" {
+  name = "test-policy"
+  description = "policy by terraform"
+  policy_content{
+    statements{
+      actions = ["ocean:deleteCluster"]
+      effect = "ALLOW"
+      resources = ["*"]
+    }
+    statements{
+      actions = ["ocean:updateCluster"]
+      effect = "DENY"
+      resources = ["*"]
+    }
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Name of the Policy.
```

Comment:
As per openAPI document it is not required parameter, did you cross-verify through postman?

Reply:
yes